src/test/java/com/project/parkinglot/service/parking_area/impl/ParkingAreaServiceImplTest.java

```diff
@@ -0,0 +1,100 @@
+package com.project.parkinglot.service.parking_area.impl;
+
+import com.project.parkinglot.base.BaseServiceTest;
+import com.project.parkinglot.builder.model.entity.ParkingAreaEntityBuilder;
+import com.project.parkinglot.exception.parkingarea.ParkingAreaNotFoundException;
+import com.project.parkinglot.model.ParkingArea;
+import com.project.parkinglot.model.entity.ParkingAreaEntity;
+import com.project.parkinglot.repository.ParkingAreaRepository;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+
+import java.util.Optional;
+import java.util.UUID;
+
+public class ParkingAreaServiceImplTest extends BaseServiceTest {
```

Comment:
ParkingAreaControllerTest'de getParkingAreaById için Integration Test yazılması gerekiyor.